examples/flutter_gallery/lib/demo/customized/initial_welcome.dart

```diff
@@ -0,0 +1,145 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'animation_helpers.dart';
+
+const String _kGalleryAssetsPackage = 'flutter_gallery_assets';
+const int _kAnimationDuration = 1200;
+
+class InitialWelcome extends StatefulWidget {
+  const InitialWelcome({ Key key }) : super(key: key);
+
+  @override
+  InitialWelcomeState createState() => InitialWelcomeState();
+}
+
+class InitialWelcomeState extends State<InitialWelcome> with TickerProviderStateMixin {
+
+  AnimationController _animationController;
+  Animation<double> _heroFadeInAnimation;
+  Animation<double> _textFadeInAnimation;
+
+  @override
+  void initState() {
+    super.initState();
+    _configureAnimations();
+    animate();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Stack(
+      children: <Widget>[
+        FadeTransition(
+          opacity: _heroFadeInAnimation,
+          child: OverflowBox(
```

Comment:
not sure why this is needed. Perhaps add a code comment

Reply:
Wasn't needed so i removed it